engine-jdk8-tests/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/JavaFXClassLoadingTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.engine.valuehandling;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+import javax.validation.ValidationException;
+
+import org.fest.assertions.Assertions;
+import org.hibernate.validator.internal.util.privilegedactions.LoadClass;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.classloader.ShrinkWrapClassLoader;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Test;
+
+/**
+ * @author Davide D'Alto
+ */
+public class JavaFXClassLoadingTest {
+
+	/**
+	 * This class will be present in the TCCL beacuse is part of JDK 8
+	 */
+	private static final String JAVAFX_APPLICATION_CLASS = "javafx.application.Application";
```

Comment:
That's because if nothing sets explicitly the TCCL, the default is that the TCCL is set to the system classpath, right?